app/lib/features/chat/widgets/custom_input.dart

```diff
@@ -1,15 +1,17 @@
 import 'dart:io';
```

Comment:
@kumarpalsinh25 I'd like for you to also take a look at this. I don't know this code part enough but you worked on the chat attachment selection and it would be a shame if that broke now ....

Reply:
I have tested this part of code, and it works same regardless. Still, @kumarpalsinh25 can review it for reassurance.